### PR TITLE
Update model link in Puebla-Nahuatl

### PR DESCRIPTION
**Pull request by ftshijt**
1. update symlink to conformer
2. default use specaug and sp

### CODE REVIEW

**Comment by ftshijt** [comment]
Update yolo-mixtec model in this PR as well.

This PR is able to merge from my side.

File: egs/yoloxochitl_mixtec/asr1/RESULTS.md [hunk 1]
```diff
@@ -1,36 +1,131 @@
-# RESULTS (100 epoch using single GPU)
+<!-- Generated by /export/c04/jiatong/tools/espnet_update/espnet/utils/show_result.sh -->
```
Comment (sw005320) [inline egs/yoloxochitl_mixtec/asr1/RESULTS.md:1]: Can you put the newer results at the top?

File: egs/puebla_nahuatl/asr1/RESULTS.md [hunk 2]
```diff
@@ -42,7 +42,7 @@
 ## train_elan_p_272_up_466_reserve_sp_pytorch_elan_p_272_up_466_reserve-conformer-specaug-sp
```
Comment (sw005320) [inline egs/puebla_nahuatl/asr1/RESULTS.md:42]: Can you put the newer (better) result at the top?

**Comment by sw005320** [comment]
Can you also mention that our activities for endangered languages in the main `README.md`?
Maybe one-sentence description in https://github.com/espnet/espnet#asr-automatic-speech-recognition?

**Comment by ftshijt** [comment]
> Can you also mention that our activities for endangered languages in the main `README.md`?
> Maybe one-sentence description in https://github.com/espnet/espnet#asr-automatic-speech-recognition?

I'm not sure if the style is OK, how about like this:

`- ASR examples for boosting endangered language documentation (Please refer to egs/puebla_nahuatl and egs/yoloxochitl_mixtec for details)`

**Comment by sw005320** [comment]
maybe supporting instead of boosting?

**Comment by sw005320** [comment]
Many thanks!

**Pull request merged**